nuxeo-retention/src/main/java/org/nuxeo/retention/listeners/RetentionBusinessEventListener.java

```diff
@@ -60,19 +62,12 @@ public void handleEvent(Event event) {
                  .append(RetentionConstants.STARTING_POINT_EVENT_PROP)
                  .append(" = ")
                  .append(NXQL.escapeString(eventName));
-            if (StringUtils.isBlank(eventInput)) {
```

Comment:
Why enventInput is not needed anymore to build the query ?

Reply:
`eventInput` can be a value that needs to be injected in the expression ctx that we want to evaluate. something you obviously can't do within NXQL